### PR TITLE
Todo cleanup

### PR DESCRIPTION
**Pull request by wackerow**
https://www.notion.so/efdn/TODOs-Cleanup-9353e3c12f144fe7801ce5e2d3950d9d

A start at handling a few of the low-hanging TODO's accumulating in the codebase. This does not cover all of them. Many TODO's have been broken off as their own actions (linked to within above doc), a few others could use a brief discussion as to how to handle.

- Removes some unused/redundant classes in `Project` and `ProjectProfile`
- Removes `furthestStep` todo, as the `FormProgressWidget` is no longer utilized by the BrightId verification flow
- Extracts overlapping `.tag` properties into shared class to limit redundancy throughout codebase
- Uses our dynamic chain label for `RecipientSubmissionWidget` instead of hardcoding L2 name.
- Removes `formatDateFromNow` as its usage has been replaced with `TimeLeft` component.

### CODE REVIEW

File: vue-app/src/components/RecipientSubmissionWidget.vue [hunk 1]
```diff
@@ -209,6 +209,11 @@ export default class RecipientSubmissionWidget extends Vue {
     return '-'
   }
 
+  get chainLabel(): string {
+    const chain = CHAIN_INFO[Number(process.env.VUE_APP_ETHEREUM_API_CHAINID)]
+    return chain.label
+  }
```
Comment (pettinarip) [inline vue-app/src/components/RecipientSubmissionWidget.vue:215]: Nice

**Review by pettinarip — APPROVED** [review]
Gj in tackling those todos.

**Review by corwintines — APPROVED** [review]
Nice!

**Review by samajammin — COMMENTED** [review]
👍

**Pull request merged**